cfpb/cfgov-refresh
# Add ability to output filterable list items as 50/50 info units

In order to achieve the desired filterable list of stories on the soon-to-be-more-populated Everyone Has a Story, which is to be moved to Wagtail next week, this PR adds an option to the Filter Controls (aka Filterable List Controls) module that tells it to output the results in a 50/50 info unit format.

Refer to this mockup: https://[GHE]/storage/user/368/files/3bb37a72-557d-11e6-90da-4cecb98acab2

**We would really like to get this in tomorrow's code freeze, if possible, so a timely review would be appreciated.**

## Additions

- Ability to output filterable list items as 50/50 info units

## Testing

1. Pull branch
2. Migrate
3. Run server
4. Create a new Sublanding Filterable page and add a Filter Controls module, checking the **Render preview items as 50-50s** option
5. Add a child Learn or Doc Detail page or two to that Sublanding Filterable. For each that you add, be sure to put stuff in the **Page Preview Fields** in the Configuration tab.
6. Preview the Sublanding Filterable and see the filter list items showing up as 50/50s.

## Screenshots

![screen shot 2016-12-07 at 15 47 08](https://cloud.githubusercontent.com/assets/1044670/20985971/b560fd12-bc94-11e6-8469-10436343515f.png)

⬇️ 

![screen shot 2016-12-07 at 15 10 10](https://cloud.githubusercontent.com/assets/1044670/20985979/bd7b8f9e-bc94-11e6-9b9a-7b911e6860ab.png)


## Notes

This is an MVP which uses the info unit macro directly. Consequently, it doesn't do the following:
- Display the category of the item
  - This would ideally be addressed by adding to the info unit macro the ability to display a passed-in category value.
  - As a workaround, it could be noted in the Preview Description's Rich Text Field.
- Automatically generate a "play" icon over the images that lead to video stories.
  - This would ideally be addressed by adding to the info unit macro the ability to add such an icon to the image, if some boolean was passed in that turned it on -- perhaps by setting a tag on the item?
  - As a workaround, it could be added to the preview image before uploading.
- Not requested explicitly, but given the recent A/B testing results, I suspect we might want to link the images?

/cc @keelerr @elizbond

## Checklist
